### ISSUE TITLE
Docker Desktop 4.13.1 (90346)  startup is blocked for some reasons

### ISSUE THREAD

**Issue by Seshiria** [docker/for-win#13051]
- [X] I have tried with the latest version of Docker Desktop
  - [ ] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID:

### Actual behavior
Because I recently changed the system hardware, and then run Docker Desktop, the following prompt appears:
![photo_2022-11-05_14-36-03](https://user-images.githubusercontent.com/57834968/200112644-128ffcbe-8316-4b21-bc4c-3eb42f2b28c8.jpg)

After I delete “<HOME>\AppData\Roaming\Docker\locked-directories” according to the log file, the Docker Desktop continues to start successfully.At that time, the “<HOME>\AppData\Roaming\Docker\locked-directories” file content was blank.
[log.0.txt](https://github.com/docker/for-win/files/9943458/log.0.txt)

It's not clear whether this will lead to other consequences, so I reported this problem.

### Information

  - Windows Version: windows 11 22621.755
  - Docker Desktop Version: Docker Desktop 4.13.1 (90346) 
  - WSL2 or Hyper-V backend?  wsl2
  - Are you running inside a virtualized Windows e.g. on a cloud server or a VM:no

### Output of `& "C:\Program Files\Docker\Docker\resources\com.docker.diagnose.exe" check`

Please investigate the following 1 issue:

1 : The test: are the backend processes running?
    Failed with: 1 error occurred:
        * vpnkit.exe is not running

### Steps to reproduce the behavior

  1. ...
  2. ...

**Comment by josefsabl** [comment]
This actually resolved the issue: https://github.com/docker/for-win/issues/13049#issuecomment-1303035087

**Comment by Seshiria** [comment]
> This actually resolved the issue: [#13049 (comment)](https://github.com/docker/for-win/issues/13049#issuecomment-1303035087)

I checked the “locked-directories” file when it could not be started. It was a blank file of 0B, and there was nothing in it.

I think it's caused by additional problems,

**Comment by Seshiria** [comment]
After updating to Docker Desktop 4.14.0 (91374), com.docker.diagnose The exe check error has been eliminated.

````
[PASS] DD0027: is there available disk space on the host?
[PASS] DD0028: is there available VM disk space?
[PASS] DD0002: does the bootloader have virtualization enabled?
[SKIP] DD0018: does the host support virtualization?
[PASS] DD0001: is the application running?
[PASS] DD0022: is the Virtual Machine Platform Windows Feature enabled?
[PASS] DD0021: is the WSL 2 Windows Feature enabled?
[PASS] DD0024: is WSL installed?
[PASS] DD0025: are WSL distros installed?
[PASS] DD0026: is the WSL LxssManager service running?
[PASS] DD0029: is the WSL 2 Linux filesystem corrupt?
[PASS] DD0035: is the VM time synchronized?
[PASS] DD0017: can a VM be started?
[PASS] DD0016: is the LinuxKit VM running?
[PASS] DD0011: are the LinuxKit services running?
[PASS] DD0004: is the Docker engine running?
[PASS] DD0015: are the binary symlinks installed?
[PASS] DD0031: does the Docker API work?
[PASS] DD0013: is the $PATH ok?
[PASS] DD0003: is the Docker CLI working?
[PASS] DD0005: is the user in the docker-users group?
[PASS] DD0014: are the backend processes running?
[PASS] DD0007: is the backend responding?
[PASS] DD0008: is the native API responding?
[PASS] DD0009: is the vpnkit API responding?
[PASS] DD0010: is the Docker API proxy responding?
[PASS] DD0006: is the Docker Desktop Service responding?
[PASS] DD0012: is the VM networking working?
[SKIP] DD0030: is the image access management authorized?
[PASS] DD0033: does the host have Internet access?
[PASS] DD0002: does the bootloader have virtualization enabled?
[PASS] DD0018: does the host support virtualization?
[PASS] DD0001: is the application running?
[PASS] DD0022: is the Virtual Machine Platform Windows Feature enabled?
[PASS] DD0021: is the WSL 2 Windows Feature enabled?
[PASS] DD0024: is WSL installed?
[PASS] DD0025: are WSL distros installed?
[PASS] DD0026: is the WSL LxssManager service running?
[PASS] DD0029: is the WSL 2 Linux filesystem corrupt?
[PASS] DD0035: is the VM time synchronized?
[PASS] DD0017: can a VM be started?
[PASS] DD0016: is the LinuxKit VM running?
[PASS] DD0011: are the LinuxKit services running?
[PASS] DD0004: is the Docker engine running?
[PASS] DD0015: are the binary symlinks installed?
[PASS] DD0031: does the Docker API work?
[PASS] DD0032: do Docker networks overlap with host IPs?
No fatal errors detected.
````
